### ISSUE TITLE
Use PackageReference deletes projects

### ISSUE THREAD

**Issue by CarlosOnline** [0UserName/NuGetSwitcher#6]
Repro:
1. Use ProjectReference
2. Use PackageReference

Packages are deleted.

**Comment by jz4M** [comment]
I have a similar problem, where it removes projects from the solution.

**Comment by 0UserName** [comment]
Hi. What version of VS are you using? Can you prepare a mini-project for reproduction?

**Comment by jkappel71** [comment]
Yes, I have see the same bug.  I don't think you need a sample project to reproduce.  What I have seen is that if you press the "Use Project Reference" button more than once (which accidentally happens sometimes) then click "Use Package Reference".  When you re-load the solution some projects are no longer there.

I am using Visual Studio 2022.

**Comment by brandonleetran** [comment]
What this issue ever resolved?

**Comment by 0UserName** [comment]
This project is small and open. You can make PR at any time.
